src/libcollections/btree/map.rs

```diff
@@ -1465,6 +1465,13 @@ impl<'a, K: Ord, V> Entry<'a, K, V> {
 }
 
 impl<'a, K: Ord, V> VacantEntry<'a, K, V> {
+    /// Gets a reference to the key that would be used when inserting a value
+    /// through the VacantEntry.
+    #[unstable(feature = "map_entry_keys", issue = "1541")]
```

Comment:
I think this will have to reference an issue in the rust-lang/rust repo, not rust-lang/rfcs.